src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/config/HmppsNonAssociationsApiExceptionHandler.kt

```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.hmpps.hmppsnonassociations.config
+package uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.config
```

Comment:
This was a sneaky one, because of a different package name (even tho it was in the correct directory) the exception handling was not working.

Reply:
Nice spot